packages/database/src/TupaiaDatabase.js

```diff
@@ -411,6 +411,9 @@ export class TupaiaDatabase {
    * Runs an arbitrary SQL query against the database.
    *
    * Use only for situations in which Knex is not able to assemble a query.
+   *
+   * @template Result
+   * @returns {Promise<Result>} execution result
```

Comment:
This allows typescript to treat this function as a generic function, which really improves things!